README.md

```diff
@@ -82,14 +82,16 @@ usage: buddha [flags] job_file jobs...
 flags:
   --config-dir=/etc/buddha.d  global job configuration directory
   --config=<file>             manually specify job configuration file
-  --stdin                     accept job configuration from STDIN
+  --stdin                     accept job configuration from stdin
   --version                   display version information
+  -y                          confirm run all
+  -h, --help                  shows this help
 
 examples:
   to invoke api_server from /etc/buddha.d:
     $ buddha api_server
   to invoke all jobs from /etc/buddha.d:
-    $ buddha all
+    $ buddha -y all
```

Comment:
Until recently is was "in inode order" :p

As a user I just want to type `buddha` and have all my processes restarted if they are not on the latest code, isn't that the goal of the "all" option ? Each process would have a "before" that would abort if it's already running the latest version. Then I don't have to maintain the state of which machine and process is on what version and can run a simple `for machine in $(my_machines); do ssh $machine buddha -y all; done`, and restart the command if anything fails like my connectivity.